mail/mlmmj/Portfile

```diff
@@ -0,0 +1,30 @@
+PortSystem          1.0
+
```

Comment:
We need this here:
```
PortGroup legacysupport 1.1
```

Reply:
Since you want this due to `strndup`, it should be limited to those platforms that don't have `strndup` (i.e. Mac OS X 10.6 and earlier):

```
# strndup
legacysupport.newest_darwin_requires_legacy 10
```

Reply:
Actually I can't reproduce this situation because I don't have legacy Mac systems.

Reply:
sure, but then just follow the guidance of people who do/did reproduce it and tell you how to fix it. Please make the suggested change.

Reply:
Yes, now it is incorporated by this commit 68ae435 . Thank you for the suggession!